cppcache/integration/test/PartitionRegionOpsTest.cpp

```diff
@@ -144,9 +146,9 @@ void verifyMetadataWasRemovedAtFirstError() {
       }
     }
   }
-  ASSERT_TRUE((timeoutErrors == metadataRemovedDueToTimeout) &&
-              (ioErrors == metadataRemovedDueToIoErr) &&
-              (metadataRemovedDueToTimeout != metadataRemovedDueToIoErr));
+  ASSERT_EQ(timeoutErrors, metadataRemovedDueToTimeout);
```

Comment:
Alternatively, if you want to see the results of all three checks and fail if any of them fail you can use `EXPECT_*` macros. There are also more options for expecting and asserting in the `gmock` module.

Reply:
Thanks, I did not know.